package.json

```diff
@@ -122,6 +122,9 @@
     "run-script-os": "^1.1.6",
     "typescript": "~4.5.5"
   },
+  "engines": {
```

Comment:
Should we do this to all the package.json?

Reply:
If we want to force users of our packages to use specific versions, maybe as a future change. For now though I just want to ensure none of our devs (or CI) is on an older version, which this should accomplish in practice.

Reply:
How about the other mono repos (server and azure)